analysis/common_variables_sensitivity.py

```diff
@@ -57,7 +33,7 @@
 
         
     ),
-    cov_bin_anticoagulants_4mofup_bnf = patients.with_these_medications(
+    sub_bin_anticoagulants_sensitivity_bnf = patients.with_these_medications(
         anticoagulants_bnf,
         returning = 'binary_flag',
         between = ["discharge_date" ,"end_date_outcome" ],
```

Comment:
The analysis plan states "Sensitivity analysis – analysis restricted to participants with at least 4-months follow-up post covid-19 infection, and subgroup by whether prescribed anticoagulants 4-months after discharge, or not.  Analysis restricted to hospitalised subgroup. " so I think this variable needs to be pulled between discharge_date + 4 months and end_data_outcome. For consistency, I would define 4 months as 112 days.